src/main/java/jmgomez/apipolicy/config/SecurityConfig.java

```diff
@@ -21,23 +21,29 @@ public class SecurityConfig {
 
     @Bean
     public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Exception {
-        http.csrf(AbstractHttpConfigurer::disable)
-                .authorizeHttpRequests(authorize -> authorize
-                        .requestMatchers("/policies").permitAll()
-                        .anyRequest().authenticated())
-                .httpBasic(Customizer.withDefaults());
-    return http.build();
+        return http.csrf(csrf -> csrf.disable())
+                .authorizeHttpRequests(authorize -> {
+                    authorize
+                            .requestMatchers("/policies").authenticated()
+                            .requestMatchers("/policies/**").authenticated();
+                })
+                .sessionManagement(session -> session
+                        .sessionCreationPolicy(SessionCreationPolicy.STATELESS))
+                .httpBasic(Customizer.withDefaults())
+                .build();
     }
 
     @Bean
     public UserDetailsService userDetailsService() {
                 UserDetails user1 = User.withUsername("00000000T")
                 .password(passwordEncoder().encode("12345user1"))
+                .authorities("read")
```

Comment:
are you using this for anything?

Reply:
No, it was a try when I was implementing authorization